assets/src/exit-modal-survey/index.js

```diff
@@ -1,7 +1,6 @@
 /**
  * External dependencies
  */
-import React from 'react';
```

Comment:
Take note that we no longer have to import React for react related work.  This is mostly applicable where jsx is being used in the file.  However I think if we're using components of React (i.e. `Component`) it'd probably still be good to import it just to make it clear.